### PR TITLE
[ADD] multiply taxes by 100

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

**Comment by legalsylvain** [comment]
:+1: 
Thanks.

**Comment by hbrunn** [comment]
will be green after #668 is merged

**Comment by jjscarafia** [comment]
:+1:

File: addons/account/migrations/9.0.1.1/post-migration.py [hunk 1]
```diff
@@ -417,6 +417,10 @@ def migrate(env, version):
     WHERE child_depend IS True
     """)
 
+    # In v9, percentages are expressed as percentage
```
Comment (pedrobaeza) [inline addons/account/migrations/9.0.1.1/post-migration.py]: Better to say: `In v9, percentages are expressed as hundred-based percentage, not one-based percentage`

**Review by susport — COMMENTED** [review]
👍

**Pull request merged**